pyproject.toml

```
@@ -7,7 +7,7 @@ packages = ["src/abbfreeathome"]
 
 [project]
 name = "local-abbfreeathome"
-version = "1.17.4"
+version = "1.18.0"
```

Let's hold off on bumping the version just yet, I'll do that in a separate PR since we'll have multiple PR's in the next release.

```suggestion
version = "1.17.4"
```